bin/prettier.js

```diff
@@ -2,4 +2,11 @@
 
```

Comment:
Can we configure ESLint to avoid use ES featured that can not be used in old Node.js in this file?

Reply:
Shouldn't this enough? https://github.com/prettier/prettier/pull/12306/files#diff-510e5a31af1f6dbb741c0fef3a97d24c9be24f9193247dc8ed2c3753d5b54457R170

We already run test on Node.js v0.10.48